### PR TITLE
Feature/DEV37 employee page

### PR DESCRIPTION
**Pull request by AAKlimkov**
image

### CODE REVIEW

File: src/modules/users/ui/users-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    department: string;
+    position: string;
+}
+interface UserDialogProps {
+    title: string;
+    onConfirm: (formData: FormData) => void;
+    onClose: () => void;
+    initialValues: FormData;
+}
+const UserDialog = ({ title, onConfirm, onClose, initialValues }: UserDialogProps) => {
```
Comment (shev7) [inline src/modules/users/ui/users-dialog.tsx]:
```suggestion
}

const UserDialog = ({ title, onConfirm, onClose, initialValues }: UserDialogProps) => {
```

File: src/modules/users/ui/users-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import { mapDepartmentsToSelectOptions, mapPositionsToSelectOptions } from "@/helpers/convert/maps";
+
+interface FormData {
+    first_name: string;
+    last_name: string;
+    email: string;
+    password: string;
+    department: string;
+    position: string;
+}
```
Comment (shev7) [inline src/modules/users/ui/users-dialog.tsx]: use UserFormSchemaType, and please refactor in other forms

**Review by shev7 — APPROVED** [review]

**Review by savelievamareena — APPROVED** [review]

File: src/modules/users/ui/users-dialog.tsx (hunk 2 above)
Comment (AAKlimkov) [reply src/modules/users/ui/users-dialog.tsx]: fixed

File: src/modules/users/ui/users-dialog.tsx (hunk 1 above)
Comment (AAKlimkov) [reply src/modules/users/ui/users-dialog.tsx]: fixed

File: src/modules/users/ui/users-list.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    const [deleteUser] = useUserDelete();
+
+    const menuProps: Action = {
+        onDelete: (id: string) =>
+            openConfirm({
+                title: t("delete confirmation"),
+                onConfirm: () => void deleteUser({ variables: { user: { userId: id } } }),
+            }),
+
+        onUpdate: (id: string) => navigate(`/users/${id}`),
```
Comment (RC-3222) [inline src/modules/users/ui/users-list.tsx]: Template string could be replaced with `routes.users.userById(id)`

**Review by RC-3222 — APPROVED** [review]

**Pull request merged**